node/src/chain_spec/oak.rs

```diff
@@ -190,6 +191,129 @@ pub fn oak_staging() -> ChainSpec {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"OAK Network",
+		// ID
+		"oak",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../../distribution/oak_alloc.json")[..];
+			let initial_allocation: Vec<(AccountId, Balance)> =
+				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 1_000_000_000;
+			validate_allocation(
+				initial_allocation.clone(),
+				ALLOC_TOKENS_TOTAL,
+				EXISTENTIAL_DEPOSIT,
+			);
+
+			let vesting_json = &include_bytes!("../../../distribution/oak_vesting.json")[..];
+			let initial_vesting: Vec<(u64, Vec<(AccountId, Balance)>)> =
+				serde_json::from_slice(vesting_json).unwrap();
+
+			let vested_tokens = 9_419_999_999_999_999_919;
+			let vest_starting_time: u64 = 1669874400;
+			let vest_ending_time: u64 = 1764568800;
+			validate_vesting(
+				initial_vesting.clone(),
+				vested_tokens,
+				EXISTENTIAL_DEPOSIT,
+				vest_starting_time,
+				vest_ending_time,
+			);
+
+			testnet_genesis(
+				// initial collators.
+				vec![
+					(
+						// SS58 prefix 51: 67He8TSn5ayD2p2ZS9WDQtuVuW7Z8RXXvBx5NyszGwcEGpzS
+						hex!["325603af5d8d4e284646556835df92977d48c8100eaf92e110d4b75d4650a73c"]
+							.into(),
+						hex!["325603af5d8d4e284646556835df92977d48c8100eaf92e110d4b75d4650a73c"]
+							.unchecked_into(),
+					),
+					(
+						// SS58 prefix 51: 6Azd1ebort6eZUDfg9f4A75fmD1JCV6ysHaFnTEqEPEuGpbP
+						hex!["d64c03d1ab36318a0d7da660670c83928261fc2a3464f92218ff5d42561e9a40"]
+							.into(),
+						hex!["d64c03d1ab36318a0d7da660670c83928261fc2a3464f92218ff5d42561e9a40"]
+							.unchecked_into(),
+					),
+					(
+						// SS58 prefix 51: 6AQf3NV66CTNSsFuY1ehA1zJySYQybQwvC5Yq8ufqcZVQ3iN
+						hex!["bc6480f3f236dcd444809ace67e5a64bd35d7757b46d051bf300264bfc1eea55"]
+							.into(),
+						hex!["bc6480f3f236dcd444809ace67e5a64bd35d7757b46d051bf300264bfc1eea55"]
+							.unchecked_into(),
+					),
+					(
+						// SS58 prefix 51: 66AJHj124JZnpKnWFoJGx41uSVqjyUz6tmVPfZWnhemBe8wG
+						hex!["00804668906bbe7934ba89ef18c17c62c2dc175ca8df33621f2413e677c7144f"]
+							.into(),
+						hex!["00804668906bbe7934ba89ef18c17c62c2dc175ca8df33621f2413e677c7144f"]
+							.unchecked_into(),
+					),
+				],
+				// 66MGxr9zcyJ6ka6FBQmT1VSvMqARKfwBT7589Fikii1Ci5sg
+				hex!["08df8338e854d8d589dedd4305c11e589cbef994e5dd00c7bb8fb7d277705b06"].into(),
+				initial_allocation,
+				REGISTERED_PARA_ID.into(),
+				vec![
+					b"AutomationTime".to_vec(),
+					b"Balances".to_vec(),
+					b"Bounties".to_vec(),
+					b"Currencies".to_vec(),
+					b"Democracy".to_vec(),
+					b"ParachainStaking".to_vec(),
+					b"PolkadotXcm".to_vec(),
+					b"Treasury".to_vec(),
+					b"XTokens".to_vec(),
```

Comment:
is this the right capitalization? `XTokens` vs `xTokens`?

Reply:
yah. naming comes from runtime files: https://github.com/OAK-Foundation/OAK-blockchain/blob/master/runtime/oak/src/lib.rs#L1083